tests/templates/kuttl/resources/10-assert.yaml.j2

```diff
@@ -19,26 +19,20 @@ spec:
         requests:
           cpu: 250m
           memory: 512Mi
-{% if lookup('env', 'VECTOR_AGGREGATOR') %}
-    - name: vector
-{% endif %}
 ---
 apiVersion: v1
 kind: Pod
 metadata:
   name: resources-crd-driver
 spec:
   containers:
-{% if lookup('env', 'VECTOR_AGGREGATOR') %}
-    - name: vector
-{% endif %}
     - name: spark
       resources:
         limits:
           cpu: "1"
           memory: 1Gi
         requests:
-          cpu: 200m
+          cpu: "1"
```

Comment:
Sorry for the dump question, but why did this change?

Reply:
No, it was surprising to me too. It's because of [this](https://github.com/stackabletech/spark-k8s-operator/blob/ca32d38ebc6cac6852a5b1ac2289cd5c576fd8de/rust/crd/src/lib.rs#L757).

Those properties take precedence over the pod templates.

Reply:
Ahh forgot about that. Could we please add a comment like
`This resources are set from spark-submit according to the Spark config`